staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4684,6 +4684,10 @@ const (
 	// HugePages request, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	// As burst is not supported for HugePages, we would only quota its request, and ignore the limit.
 	ResourceRequestsHugePagesPrefix = "requests.hugepages-"
+	// Default resource requests prefix
+	DefaultResourceRequestsPrefix = "requests."
+	// Default resource limits prefix
+	DefaultResourceLimitsPrefix = "limits."
```

Comment:
its not clear why we need this if we only support quota of the form "requests.[resourceName]"